jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -76,33 +76,6 @@ def _warn_if_not_found(app, file_lists):
     return found
 
 
-def _parse_as_image(path):
-    """
-    Parse and load a 2D image. ``CCDData`` objects require a unit be defined
-    in the fits header - if none is provided, use a fallback and
-    raise an error.
-    """
-    with fits.open(path) as hdulist:
-
-        header = hdulist[0].header.copy()
-        meta = dict(header)
-
-        wcs = WCS(header)
-
-        try:
-            image_ccd = CCDData.read(path, wcs=wcs)
-        except ValueError as e:
-            if str(e) == "a unit for CCDData must be specified.":
-                logging.warning("No 'BUNIT' defined in the header, using 'Jy'.")
```

Comment:
I think I might have accidentally fixed #771 here. Though I don't know if Mosviz will choke on the way Imviz handles invalid units or not. Someone should check.

https://github.com/spacetelescope/jdaviz/blob/d0c2ce00041101e636ed71c97e7eb00047ba96df/jdaviz/configs/imviz/plugins/parsers.py#L270-L273